auto_nag/scripts/assignee_no_login.py

```diff
@@ -67,15 +76,23 @@ def handle_bug(self, bug, data):
         else:
             do_needinfo = True
 
-        if do_needinfo and not self.add_auto_ni(
-            bugid,
-            {
-                "mail": bug["triage_owner"],
-                "nickname": bug["triage_owner_detail"]["nick"],
-            },
-        ):
-            # Don't unassign if we can't needinfo.
-            return None
+        if do_needinfo:
+            if not self.add_auto_ni(
+                bugid,
+                {
+                    "mail": bug["triage_owner"],
+                    "nickname": bug["triage_owner_detail"]["nick"],
+                },
+            ):
+                # Don't unassign if we can't needinfo.
+                return None
+        else:
+            if (
+                self.unassign_count[bug["triage_owner"]]
+                >= self.max_unassign_per_triage_owner
+            ):
+                return None
+            self.unassign_count[bug["triage_owner"]] += 1
```

Comment:
Do we need to handle cases where a component has no triage owner?

Reply:
In theory yes, we should check if the triage owner exists and is active, but we can rely on the missing triage owner tool to help us always having a triage owner.